components/PriceGraph/BazaarPriceGraph/PriceGraphConfigSingle.tsx

```diff
@@ -0,0 +1,244 @@
+                symbol: 'none',
+                yAxisIndex: 0,
+                tooltip: {
+                    show: true,
+                    valueFormatter: value => `${numberWithThousandsSeperators(value)} Coins`
+                },
+                data: []
+            },
+            {
+                name: 'Volume (buy)',
+                type: 'line',
+                color: '#4B0082',
+                symbol: 'none',
+                yAxisIndex: 1,
+                tooltip: {
+                    show: true,
+                    valueFormatter: value => `${numberWithThousandsSeperators(value)}`
+                },
+                data: []
+            },
+            {
+                name: 'Moving (buy)',
+                type: 'line',
+                symbol: 'none',
+                yAxisIndex: 1,
+                tooltip: {
+                    show: true,
+                    valueFormatter: value => `${numberWithThousandsSeperators(value)}`
+                },
+                data: []
+            },
+            {
+                name: 'Price (sell)',
+                type: 'k',
+                color: '#32CD32',
+                smooth: true,
+                symbol: 'none',
+                lineStyle: {
+                    width: 4
+                },
+                yAxisIndex: 0,
+                tooltip: {
+                    show: true,
+                    valueFormatter: value => {
+                        if (!value || (value && value.length === 0)) {
+                            return ''
+                        }
+                        return `${numberWithThousandsSeperators(value)} Coins`
+                    }
+                }
+            },
+            {
+                name: 'Min (sell)',
+                type: 'line',
+                color: '#228B22',
+                smooth: true,
+                symbol: 'none',
+                yAxisIndex: 0,
+                tooltip: {
+                    show: true,
+                    valueFormatter: value => `${numberWithThousandsSeperators(value)} Coins`
+                },
+                data: []
+            },
+            {
+                name: 'Max (sell)',
+                type: 'line',
+                color: '#B22222',
+                smooth: true,
+                symbol: 'none',
+                yAxisIndex: 0,
+                tooltip: {
+                    show: true,
+                    valueFormatter: value => `${numberWithThousandsSeperators(value)} Coins`
+                },
+                data: []
+            },
+            {
+                name: 'Volume (sell)',
+                type: 'line',
+                color: '#4B0082',
+                smooth: true,
+                symbol: 'none',
+                yAxisIndex: 1,
+                tooltip: {
+                    show: true,
+                    valueFormatter: value => `${numberWithThousandsSeperators(value)}`
+                },
+                data: []
```

Comment:
this is identical with the settings above and below, why isn't this a variable of some sorts?
maybe just a template expanded with the ... operator

Reply:
Dont really know what you mean with a template being extended with `...`, but I think I refactored it good enough :)